lib/install/management/store_files.go

```diff
@@ -187,9 +187,9 @@ func (d *Dispatcher) deleteFilesIteratively(m *object.DatastoreFileManager, ds *
 func (d *Dispatcher) deleteVMFSFiles(m *object.DatastoreFileManager, ds *object.Datastore, dsPath string) error {
 	defer trace.End(trace.Begin(dsPath, d.op))
 
-	for _, ext := range []string{"-delta.vmdk", "-flat.vmdk"} {
+	for _, ext := range []string{"-delta.vmdk", "-flat.vmdk", "-sesparse.vmdk"} {
 		if strings.HasSuffix(dsPath, ext) {
-			// Skip backing files as Delete() will do so via DeleteVirtualDisk
+			// Skip delete of snapshot files, as Delete() will remove all snapshot files when deleting the vmdk
```

Comment:
snapshot is a somewhat misleading term as it's specific terminology in vSphere and we do not use snapshots as such. We're attempting to avoid deleting the actual backing file (*.vmdk is a textual descriptor file that references the backing file).